static/app/components/events/interfaces/request/index.spec.tsx

```diff
@@ -53,48 +53,50 @@ describe('Request entry', function () {
               url: null,
               query: null,
               data: {
-                a: {
-                  '': {
-                    rem: [['organization:0', 's', 0, 0]],
-                    len: 1,
-                    chunks: [
-                      {
-                        type: 'redaction',
-                        text: '',
-                        rule_id: 'organization:0',
-                        remark: 's',
-                      },
-                    ],
+                0: {
```

Comment:
This test was in correct, should have had a 0 index here. the new logic causes it to fail (as it should in the old state)